.npmignore

```diff
@@ -3,17 +3,15 @@
 # See https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package
 
 client/js/bundle.vendor.js.map
+client/js/lounge.js
+client/js/libs/
```

Comment:
This will not break source maps?
Either way I'm not sure if we should ignore them or not, but I don't have a compelling argument.